content/en/blog/_posts/2022-12-06-fsgroup-on-mount.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Providing pod's FSGroup to CSI drivers during mount"
+date: 2022-12-06
+slug: kubernetes-12-06-fsgroup-on-mount
+---
+
+**Author:** Fabio Bertinatto (Red Hat), Hemant Kumar (Red Hat)
+
+We are happy to announce that the delegation of fsGroup to CSI Drivers feature has graduated to General Availability (GA) in Kubernetes 1.26.
```

Comment:
Can you add when this feature was introduced as alpha, moved to beta?  Are there any blogs from previous releases related to this feature that can be referenced here?